From db4570a60ca0085d16cc8505fbabe90dee69f617 Mon Sep 17 00:00:00 2001
From: Randy <rrandymiller@gmail.com>
Date: Sun, 9 Nov 2025 21:30:31 -0500
Subject: [PATCH] changed 2 lines firstFile

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index f00c965..bb076e7 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -6,5 +6,5 @@
 6
 7
 8
-9
-10
+-9
+-10
-- 
2.43.0

